docker/solr/precreate

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# This is a copy of solr 6.6's docker precreate script with some tweaks to get
+# the settings we want
+set -e
+
+echo "Executing $0 $@"
+
+if [[ "$VERBOSE" = "yes" ]]; then
+    set -x
+fi
+
+. /opt/docker-solr/scripts/run-initdb
+
+CORE=${1:-openoni}
+CONFIG_SOURCE=${2:-'/opt/solr/server/solr/configsets/data_driven_schema_configs'}
+if [[ -z $SOLR_HOME ]]; then
+    coresdir="/opt/solr/server/solr/mycores"
+    mkdir -p $coresdir
+else
+    coresdir=$SOLR_HOME
+fi
+coredir="$coresdir/$CORE"
+if [[ ! -d $coredir ]]; then
+    cp -r $CONFIG_SOURCE/ $coredir
+    touch "$coredir/core.properties"
+    echo created "$CORE"
+else
+    echo "core $CORE already exists"
+fi
+
+# Tweak the schema; ONI's schema is very explicit per field, so the dynamic
+# configs are no good for us :(
+rm -f $coredir/conf/managed-schema
```

Comment:
We should be able to post JSON to the managed-schema to add the fields we want, and / or provide a schema template to build it from.  That's what we do at UNL and it works very well.  I could probably help build that, or at least send over notes and our scripts, etc.